doc/api/async_context.md

```diff
@@ -136,6 +136,50 @@ changes:
 Creates a new instance of `AsyncLocalStorage`. Store is only provided within a
 `run()` call or after an `enterWith()` call.
 
+### Static method: `AsyncLocalStorage.bind(fn)`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+> Stability: 1 - Experimental
+
+* `fn` {Function} The function to bind to the current execution context.
+
+Binds the given function to the current execution context.
```

Comment:
If I read this I would expect the passed function gets somehow blessed with the current context.
Please add that this function returns a new function to replace the original.